kura/org.eclipse.kura.container.provider/src/main/java/org/eclipse/kura/container/provider/ContainerInstance.java

```diff
@@ -425,4 +445,42 @@ public State onDisabled() {
 
     }
 
-}
\ No newline at end of file
+    private void getDigestFromSignatureVerification(Map<String, Object> properties) {
```

Comment:
This could be personal taste but I think this method is hiding quite a lot of things... for starters it's not actually only `get`ting the digest, it is also updating the properties **and** the snapshot.

Personally I would make it return the digest and I would perform the properties/snapshot update in the caller method. This would render this method useless though...

At a minimum we need to update the name. Let's discuss this offline though.